metricbeat/module/traefik/health/health_test.go

```diff
@@ -30,6 +30,8 @@ import (
 
 	"github.com/elastic/beats/libbeat/common"
 	mbtest "github.com/elastic/beats/metricbeat/mb/testing"
+
+	_ "github.com/elastic/beats/metricbeat/module/traefik"
```

Comment:
To maintain the current approach, there's no need to test the metricsets individually. Right now, entire modules are tested, I'm not fully sure if to test metricsets individually will be really neccessary in the future.

The question is: what do you think about moving this file to the Traefik module folder and to have a line for each metricset, for example:

```go
func TestData(t *testing.T) {
	mbtest.TestDataFiles(t, "traefik", "health", mbtest.TestDataConfig(t))
	mbtest.TestDataFiles(t, "traefik", "another_metricset", mbtest.TestDataConfig(t))
}
```

Or even
```go
func TestData(t *testing.T) {
	mbtest.TestDataFiles(t, "traefik", mbtest.TestDataConfig(t))
}
```

This way, we'll have less testing files that maybe just have boilerplate code

Reply:
I did it intentionally, I'd also like `go test ./metricbeat/module/somemodule/somemetricset` to run these tests.
Also, I'd prefer that if `go test ./metricbeat/module/somemodule` is run, only the tests of the module are executed and not the ones of their metricsets. For that `/...` can be appended.
In general, I'd like the tests to cover the package where they are.